pkg/fanal/secret/builtin-rules.go

```diff
@@ -158,6 +158,15 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`glpat-[0-9a-zA-Z\-\_]{20}`),
 		Keywords: []string{"glpat-"},
 	},
+	{
+	        ID:              "hugging-face-access-token",
+	        Category:        types.SecretRuleCategory("HuggingFace"),
+	        Severity:        "CRITICAL",
+	        Title:           "Hugging Face Access Token",
+	        Regex:           MustCompile(`hf_[A-Za-z0-9]{39}`),
```

Comment:
Can we add any link to documents with this secret format?

Reply:
e.g. https://huggingface.co/docs/hub/en/security-tokens

Reply:
When you say add links, where should they be added specifically?

Reply:
added in https://github.com/aquasecurity/trivy/pull/6236/commits/5bc92a101e9ce54b62ffac2fbe8fbc59f3c9e3c4